modules/static/bufbuild/confluent/initref

```diff
@@ -1 +1 @@
-18c83a31d7e710e1b8978fca46a392381bf2fa7e
+v0.1.0
```

Comment:
It's the only semver release so far: https://github.com/bufbuild/confluent-proto/releases